cmd/web.go

```diff
@@ -207,12 +207,18 @@ func runWeb(ctx *cli.Context) error {
 		m.Get("/login", user.SignIn)
 		m.Post("/login", bindIgnErr(auth.SignInForm{}), user.SignInPost)
 		// TODO: only if OpenID is enabled ?
-		m.Get("/login/openid", user.SignInOpenID)
-		m.Post("/login/openid", bindIgnErr(auth.SignInOpenIDForm{}), user.SignInOpenIDPost)
-		m.Get("/openid/connect", user.ConnectOpenID)
-		m.Post("/openid/connect", bindIgnErr(auth.ConnectOpenIDForm{}), user.ConnectOpenIDPost)
-		m.Get("/openid/register", user.RegisterOpenID)
-		m.Post("/openid/register", bindIgnErr(auth.SignUpOpenIDForm{}), user.RegisterOpenIDPost)
+		m.Combo("/login/openid").
+			Get(user.SignInOpenID).
+			Post(bindIgnErr(auth.SignInOpenIDForm{}), user.SignInOpenIDPost)
+		// TODO: only if OpenID is enabled ?
```

Comment:
This is kind of related to the way openid is enabled/disabled. Should it be in Ini ? Should it be in LoginSources ?